ios/RNSentry.m

```diff
@@ -292,6 +292,35 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
     resolve(@YES);
 }
 
+RCT_EXPORT_METHOD(captureScreenshot: (RCTPromiseResolveBlock)resolve
+                  rejecter: (RCTPromiseRejectBlock)reject)
+{
+    NSArray<NSData *>* rawScreenshots = [PrivateSentrySDKOnly captureScreenshots];
+    NSMutableArray *screenshotsArray = [NSMutableArray arrayWithCapacity:[rawScreenshots count]];
+
+    int counter = 1;
+    for (NSData* raw in rawScreenshots) {
```

Comment:
M: Can this be `nil` or 0 bytes or something? we do checks on Android but not on iOS?

Reply:
I don't know, but I would say, if it can, we should check it in `cocoa` and not add such a screenshot to the array. 

@brustolin Do you know, if [this](https://github.com/getsentry/sentry-cocoa/blob/f781ecba1e2ccfc37e2156f1a9300452c0c8f4c8/Sources/Sentry/SentryScreenshot.m#LL49C17-L49C17) can be `nil` sometimes or 0 size?

Reply:
Theoretically no. And I never saw it happening. 
Nil here is not a problem, the loop will ignore nil, but I guess you could add a check for 0 size.

Reply:
@brustolin Should I add the 0 size check in the cocoa SDK, I guess the 0 size screenshot should not be in the list.

Reply:
Good point. That would be the best solution

Reply:
- https://github.com/getsentry/sentry-cocoa/pull/2459